services/common/src/main/java/org/finra/gatekeeper/GatekeeperCommonConfig.java

```diff
@@ -128,10 +136,18 @@ public AWSSecurityTokenServiceClient awsSecurityTokenServiceClient() {
     @AutoConfigureOrder(Ordered.HIGHEST_PRECEDENCE)
     public FilterRegistrationBean userProfileFilterRegistration() {
         FilterRegistrationBean userProfileFilterRegistration = new FilterRegistrationBean();
-        if(contentSecurityPolicy != null && !contentSecurityPolicy.isEmpty()) {
-            userProfileFilterRegistration.setFilter(new UserHeaderFilter(new SSOParser(userIdHeader), contentSecurityPolicy));
-        } else {
-            userProfileFilterRegistration.setFilter(new UserHeaderFilter(new SSOParser(userIdHeader)));
+        if(gatekeeperAuthProperties.getAuthServiceType().toUpperCase().equals("HEADERS")){
```

Comment:
Would it make sense to define an enum, or at the very least a static final string for this?